Corona-tracker/README.md

```diff
@@ -0,0 +1,16 @@
+<h1 align="center"> 🌵 Go-Corona</h1>
+<hr>
+<br>
+<p>Beginner friendly project which uses basics API's and provide real time data of vaccines and corona cases.
```

Comment:
Add the use of the project and steps to run it.

Reply:
I have updated changes. Please review.